CHANGES.md

```diff
@@ -18,8 +18,13 @@ To be released.
 
 ### Bug fixes
 
+ -  Fixed a bug where `Message` had serialized peer with `BinaryFormatter`.
+    [[#1455]]
```

Comment:
How about move this section to network change?

Reply:
I moved it!